Additional clarifications of steps and expectations for authors (#80)
radicle-foundation/radworks-governance: manual.md

The ***Discussion*** phase is used to review the first draft of new proposals. This is the primary period for the community to review and provide feedback on proposals drafts. The more feedback provided during this phase allows for more detailed and refined proposals to vote on in later stages.

When preparing the proposal drafts for the _Discussion_ phase, proposal authors should:
- Use [these templates](https://github.com/radicle-foundation/radworks-governance/tree/main/templates) to create their proposals
- Use [these templates](https://github.com/radicle-foundation/radworks-governance/tree/main/templates) to create their proposals. Please see the 'Proposal Types' section below to determine which template to use. 
- Try to include as much detail as possible in this first draft of their proposals

> - Post their proposal drafts under [Proposal Drafts](https://community.radworks.org/c/governance/discussions/19) on the forum by the _second Monday_ of each cycle (see 'Proposal Cycles' for details).  

Reviewer: Should 'Proposal Cycles' link somewhere?

> - Post their proposal drafts under [Proposal Drafts](https://community.radworks.org/c/governance/discussions/19) on the forum by the _second Monday_ of each cycle (see 'Proposal Cycles' for details).  

Reviewer: Open question: should we change the deadline for posting to the first wednesday of the cycle? seems like people wait until the last moment and then complain about there not being enough time for feedback to be included so maybe we should just change the requirement

**Formal Review**

The _Formal Review_ phase serves as the final review step in the governance process before proposals are submitted on-chain. Proposals in this phase incorporate feedback from the _Discussion_ phase, and are accompanied by an off-chain Snapshot poll to gauge consensus for their advancement. All Snapshot polls must begin on **5:00pm GMT+2** the **third Monday of the month** and end at **5:00pm GMT+2** on the **following Monday**. Each Snapshot poll should have three voting options: `Yes` `No` and `Abstain`. 

During _Formal Review_, proposal authors are required to:

> - Publish an new version of their proposal on the forum that incorporates feedback from the *Discussion* phase. Please use [this template]([url](https://github.com/radicle-foundation/radworks-governance/blob/main/templates/05-Formal-Review-forum-post-header)) as guide. Proposals in this stage should be posted under the [Active Proposals](https://community.radworks.org/c/governance/proposals/18) category on the forum. The phase in the title should read [Formal Review].

Reviewer: ```suggestion
- Publish an new version of their proposal on the forum that incorporates feedback from the *Discussion* phase. Please use [this template]([url](https://github.com/radicle-foundation/radworks-governance/blob/main/templates/05-Formal-Review-forum-post-header)) as a guide. Proposals in this stage should be posted under the [Active Proposals](https://community.radworks.org/c/governance/proposals/18) category on the forum. The phase in the title should read [Formal Review].
```

> - If applicable, include a draft of the executable proposal code for the community to review in the [proposal code folder]([url](https://github.com/radicle-foundation/radworks-governance/tree/main/proposals)). Use [this template]([url](https://github.com/radicle-foundation/radworks-governance/blob/main/templates/07-proposal-code-template.md)) to draft the proposal code. 

Reviewer: ```suggestion
- If applicable, submit a draft of the executable proposal code for the community to review in the [proposal code folder]([url](https://github.com/radicle-foundation/radworks-governance/tree/main/proposals)). Use [this template]([url](https://github.com/radicle-foundation/radworks-governance/blob/main/templates/07-proposal-code-template.md)) to draft the proposal code. 
```

> - Publish a corresponding Snapshot poll to your proposal on the [Radworks Snapshot page]([url](https://snapshot.org/#/gov.radworks.eth)). To publish a Snapshot poll for Formal Review, you must be added as an 'author' in the Radworks Snapshot space. This measure is in place to prevent spam proposals. Please reach out to Shelby (shelb_ee#9785 on Discord) with the Ethereum address of the party posting your proposal.

Reviewer: Should we say: If you need the Governance Committee can submit the Snapshot poll for you, please reach out to Shelby before 3pm on Monday. or something?

- Please use [this template]([url](https://github.com/radicle-foundation/radworks-governance/blob/main/templates/08-snapshot-poll-template)) to prepare the Snapshot poll. 

> - Once live, proposal authors should include a link to the Snapshot poll as a callout at the top of their proposal drafts on the forum. Please clearly state the day and time the poll closes (e.g. Monday, _month day_ at 5pm CET / 11am ET).

Reviewer: ```suggestion
- Once live, proposal authors should include a link to the Snapshot poll as a callout at the top of their proposal drafts on the forum. Please clearly state the day and time the poll closes (e.g. Monday, [MM/DD] at 5pm CET / 11am ET).
```
